src/misc/Hooks.h

```diff
@@ -23,7 +23,7 @@ template <typename H, typename... Args> void ExecuteHooks(Args&&... args) {
     }
 }
 
-DEFINE_HOOK(ControllerRead, void(OSContPad* cont_pad));
+DEFINE_HOOK(ControllerRead, void(OSContPad* pad));
```

Comment:
not a fan of `cont_pad`, but also not the biggest fan of `pad`, `controllerPad` also seems bad

idk, probably should just stay `pad` but if anyone has better ideas that'd be cool